pkg/cmd/enclave_secrets.go

```diff
@@ -105,6 +105,7 @@ func init() {
 	}
 	enclaveSecretsCmd.Flags().Bool("raw", false, "print the raw secret value without processing variables")
 	enclaveSecretsCmd.Flags().Bool("visibility", false, "include secret visibility in table output")
+	enclaveSecretsCmd.Flags().Bool("value-type", false, "include secret value type in table output")
```

Comment:
Thoughts on exposing this publicly as `type`? I think it's implicit that it refers to the value, given that the "secret type" is referred to as "visibility".

Reply:
Hey Tom! I can get behind that. 👍